testsuite/pom.xml

```diff
@@ -28,12 +28,16 @@
         <version.wildfly10>10.0.0.Final</version.wildfly10>
         <version.wildfly10.core>2.0.10.Final</version.wildfly10.core>
         <version.wildfly10.arquillian>1.1.0.Alpha1</version.wildfly10.arquillian>
+        <version.wildfly11>11.0.0.Alpha1-SNAPSHOT</version.wildfly11>
```

Comment:
It can be replaced with `<version.wildfly11>11.0.0.Alpha1</version.wildfly11>`.
It will be first valid public value. It can lead user to realize what is going on here. Better than SNAPSHOT, which looks like a bug.